src/components/progress/progress.api.ts

```diff
@@ -0,0 +1,13 @@
+import { type HTMLAttributes } from 'react';
+
+export interface IProgressProps extends HTMLAttributes<HTMLDivElement> {
+    /**
+     * Visual variant.
+     * @default md
+     */
+    variant?: 'md' | 'sm';
```

Comment:
The property is named `size` on Figma
```suggestion
    /**
     * Size of the progress component.
     * @default md
     */
    size?: 'md' | 'sm';
```